core/src/main/java/org/elasticsearch/cluster/metadata/IndexMetaData.java

```diff
@@ -1226,6 +1228,7 @@ public static boolean isOnSharedFilesystem(Settings settings) {
      * setting for this is <code>false</code>.
      */
     public static boolean isIndexUsingShadowReplicas(Settings settings) {
+        // don't use the settings directly, not to trigger manny deprecation logging
```

Comment:
Nits: `settings` -> `setting` and `not to trigger manny` -> `to not trigger verbose` (it doesn't have to be exactly this, but at least fix `manny`)